astropy/io/misc/asdf/schemas/astropy.org/astropy/time/timedelta-1.0.0.yaml

```diff
@@ -0,0 +1,30 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/asdf/asdf-schema-1.0.0"
+id: "http://astropy.org/schemas/astropy/time/time-1.1.0"
+title: Represents an instance of TimeDelta from astropy
+description: |
+  Represents the time difference between two times.
+
+type: object
+properties:
+    jd1:
+      type: number
+      description: |
+        Value representing first 64 bits of precision
+    jd2:
+      type: number
+      description: |
+        Value representing second 64 bits of precision
+    format:
+      type: string
+      description: |
+        Format of time value representation.
+    scale:
+      type: string
+      description: |
+        Time scale of input value(s).
+      enum: [tdb, tt, ut1, tcg, tcb, tai]
```

Comment:
This seems odd: there is also a `local` timescale, and `TimeDelta` can also have a scale `None` (an "any scale you want" cop-out).

Reply:
p.s. `Time` can also have scale `local` these days.

Reply:
I thought I got this from `astropy.time.TIME_DELTA_SCALES` but it does look like I missed a few...